source/components/defining-a-component.md

```diff
@@ -81,3 +81,57 @@ example, if you have a component called `blog-post`, you would create a
 file at `app/components/blog-post.js`. If your component was called
 `audio-player-controls`, the file name would be at
 `app/components/audio-player-controls.js`.
+
+### Dynamically rendering a component
+
+The `{{component}}` helper can be used to defer the selection of a component to
+run time. The `{{my-component}}` syntax would always render the same component, whereas using the `{{component}}` helper allows swapping the component rendered on the fly.
+
+The first parameter of the helper is the name of a component to render, as a string. So if you have `{{component 'blog-post'}}`, that is just the same as just `{{blog-post}}`.
+
+The real value of `{{component}}` comes from being able to dynamically pick
+the component being rendered:
```

Comment:
Can you explain in prose what the below example is doing? Let the reader to know what to expect without having to closely parse the code.